runtime/compiler/control/CompilationRuntime.hpp

```diff
@@ -1406,6 +1409,7 @@ class CompilationInfo
       S390SupportsTM                   = 0x00010000,
       S390SupportsRI                   = 0x00020000,
       S390SupportsVectorFacility       = 0x00040000,
+      IsInShutdownMode                 = 0x00080000,
```

Comment:
I would rather have this as a separate  'bool' field rather than a bit inside a int32_t entity. The reason is that setting a bit is a read-modify-write operation which needs to be done atomically. Because not all 'set' operations for these bits are performed under the same lock we cannot guarantee atomicity.